keps/sig-node/2535-ensure-secret-pulled-images/README.md

```diff
@@ -129,27 +136,25 @@ use un-encrypted...
 
 This feature will not change the behavior of pod with `ImagePullPolicy` `Always`.
 
+Enforcing periodical repull might be important in order to, for example, be able
+to recheck image license entitlement, but is a non-goal for this enhancement.
+
 ## Proposal
 
-For alpha the kubelet will keep a list, across reboots of host and restart of
-kubelet, of container images that required authentication and a list of the
-authentications that successfully pulled the image.
-For beta an API will be considered to manage the ensure metadata.
+A new API will be created to represent metadata about image pulls being authenticated. This API
```

Comment:
This makes it sound like there will be REST API surface, which isn't what we're proposing.

I think it's helpful to start by describing what the kubelet will do:

> The kubelet will track container images that required authentication and a list of the
authentications that successfully pulled the image. This list will persist across reboots of host and restart of
kubelet.

And then go into *how* it will do it

> This will be done using a files persisted within the kubelet directory on the node.
> The files will be structured and versioned using standard config file API versioning techniques.